src/src/AdminBundle/Entity/EventType.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace AdminBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * EventType.
+ *
+ * @ORM\Table(name="event_type")
+ * @ORM\Entity(repositoryClass="AdminBundle\Repository\EventTypeRepository")
+ * @UniqueEntity(
+ *     fields={"slug"},
```

Comment:
"deleted" ti tu chyba

Reply:
A ja som myslela ze tu to netreba.

Reply:
kvoli soft delete-u to bude treba vsade. Ide totiz o to, ze ak vytvoris event, zmazes ho, vytvoris novy (tak ti uz nedovoli pouzit ten slug) co je somarina, pretoze ho ma len nejaky zmazany (nepouzivany) event.